tests/NHSD.BuyingCatalogue.Solutions.Application.UnitTests/Solutions/RoadMapGetBySolutionIdTests.cs

```diff
@@ -0,0 +1,61 @@
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using NHSD.BuyingCatalogue.Infrastructure.Exceptions;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Persistence;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Queries;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Solutions.Application.UnitTests.Solutions
+{
+    [TestFixture]
+    internal sealed class RoadMapGetBySolutionIdTests
```

Comment:
Can this class be named 'GetRoadMapBySolutionIdHandlerTests' to match the class it is testing?